src/metabase/api/pulse.clj

```diff
@@ -55,16 +57,18 @@
 
 (defendpoint PUT "/:id"
   "Update a `Pulse` with ID."
-  [id :as {{:keys [name cards channels]} :body}]
-  {name     su/NonBlankString
-   cards    (su/non-empty [su/Map])
-   channels (su/non-empty [su/Map])}
+  [id :as {{:keys [name cards channels skip_if_empty]} :body}]
+  {name          su/NonBlankString
+   cards         (su/non-empty [su/Map])
+   channels      (su/non-empty [su/Map])
+   skip_if_empty s/Bool}
   (write-check Pulse id)
   (check-card-read-permissions cards)
-  (pulse/update-pulse! {:id       id
-                        :name     name
-                        :cards    (map u/get-id cards)
-                        :channels channels})
+  (pulse/update-pulse! {:id            id
+                        :name          name
+                        :cards         (map u/get-id cards)
+                        :channels      channels
+                        :skip_if_empty? skip_if_empty})
```

Comment:
We could at some point make a ring middleware that transforms the _'s into -'s that come from the API, to get rid of underscores all across the backend.